### DISCUSSION TITLE
How to contribute?

### DISCUSSION THREAD

**Discussion by ittaboba** [antvis/G6#3393] [Q&A] [answered]
Hi,

I would contribute but I don't understand how to see graph components in dev. 

What are the commands to do so?

Thanks,

Lorenzo

**Comment by hustcc** [comment, marked as answer]
- See the CONTRIBUTING：https://github.com/antvis/G6/blob/v5/CONTRIBUTING.en-US.md
- Filter the issue with label = OSCP
